server/src/test/java/io/pravega/schemaregistry/service/SchemaRegistryServiceTest.java

```diff
@@ -720,7 +720,62 @@ public void testDeleteUsingTypeAndVersion() {
                 () -> service.deleteSchema(null, groupName, SerializationFormat.Avro.getFullTypeName(), schemaName, version).join(),
                 e -> e instanceof RuntimeException);
     }
-    
+
+    @Test
+    public void testAddCustomSchema() {
```

Comment:
Please add one more test case with empty namespace field provided as input to addSchema.

Reply:
added